cookbook/email/gmail.rst

```diff
@@ -55,33 +50,73 @@ In the development configuration file, change the ``transport`` setting to
             'password'  => 'your_gmail_password',
         ));
 
-You're done!
+If you are using the Symfony Standard Edition, it's more convenient to configure
+these parameters in ``parameters.yml``:
 
-.. tip::
+.. code-block:: yaml
 
-    If you are using the Symfony Standard Edition, configure the parameters in ``parameters.yml``:
+    # app/config/parameters.yml
+    parameters:
+        # ...
+        mailer_user:      your_gmail_username
+        mailer_password:  your_gmail_password
+
+.. configuration-block::
 
     .. code-block:: yaml
 
-        # app/config/parameters.yml
-        parameters:
-            # ...
-            mailer_transport: gmail
-            mailer_host:      ~
-            mailer_user:      your_gmail_username
-            mailer_password:  your_gmail_password
+        # app/config/config_dev.yml
+        swiftmailer:
+            transport: gmail
+            username:  %mailer_user%
+            password:  %mailer_password%
```

Comment:
Thanks for the heads up. I've updated this. Hopefully this PR can now be considered finished.